src/app/components/Navbar/Navbar.jsx

```diff
@@ -84,7 +84,7 @@ export default function App({ featureRef }) {
             as={Link}
             href="/login"
```

Comment:
Hey! @subhadeepbanerjee2003 ! Thanks for the Contribution.

All you have to do is to change the href from "/login" to "/join"
```suggestion
            href="/join"
```